core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -32,11 +32,26 @@ oppia.factory('UrlInterpolationService', [
       }
     };
 
-    var getCachePrefixedUrl = function(resourcePath) {
+    var getPrefixedUrl = function(resourcePath) {
       validateResourcePath(resourcePath);
       return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
     };
 
+    var getUrlWithSlug = function(resourcePath) {
+      validateResourcePath(resourcePath);
+      stripedPath = resourcePath.substring(1);
+      if (GLOBALS.MINIFICATION || !GLOBALS.DEV_MODE) {
+        if (!angular.isUndefined(hashes[stripedPath])) {
+          hash = hashes[stripedPath];
+          splitedPath = resourcePath.split('.');
```

Comment:
This actually works as a find-from-right, it splits the path by '.' then adds the hash as a last but one and joins as a string with '.' again.